frontend/src/components/WindSettings.jsx

```diff
@@ -20,391 +19,209 @@ const useStyles = makeStyles((theme) => ({
+                            '& .MuiOutlinedInput-root': {
+                                '& .MuiInputBase-input': {
+                                    padding: '6px 8px',
+                                },
+                            },
+                        }}
+                        variant="outlined"
+                        type="number"
+                        onChange={onChange}
+                        value={value}
+                        inputProps={inputProps}
+                        fullWidth
+                    />
+                </Tooltip>
+            </Grid>
+        </Grid>
+    );
+
+    const handleAddWindBlock = () => {
+        setIsWindSelectionEnabled(true);
+        addNewWindBlock();
+    };
+
+    addNewWindBlock = () => {
+        const newWindBlock = {
+            windType: selectedWindType,
+            windDirection: envConf.Wind.Direction,
+            windVelocity: envConf.Wind.Force,
+            fluctuationPercentage: fluctuationPercentage,
+        };
+        setWindBlocks([...windBlocks, newWindBlock]);
+    };
+
+    setWindBlockData = (index, updatedData) => {
+        const updatedWindBlocks = [...windBlocks];
+        updatedWindBlocks[index] = { ...updatedWindBlocks[index], ...updatedData };
+        setWindBlocks(updatedWindBlocks);
+    };
+
+    deleteWindBlock = (index) => {
+        const updatedWindBlocks = [...windBlocks];
+        updatedWindBlocks.splice(index, 1);
+        setWindBlocks(updatedWindBlocks);
+    };
 
-    const StyledSelect = styled(Select)(({ theme }) => ({
-        backgroundColor: '#F5F5DC',
-            '& .MuiInputBase-input': {
-                padding: '6px 8px',
-                height: '1em',
-            } 
-    }));
 
     return (
-        <Grid container spacing={5} direction="column" classes={{ root: classes.transparentBackground }} >
-            <Grid item container spacing={2} xs={12} classes={{ root: classes.backdropFilter, zIndex: 2 }} >
-                <Grid item container alignItems="center" direction="row">
-                    <Grid item xs={4}>
-                        <InputLabel id="WindType" sx={{ marginRight: 2, width: '200px', flexShrink: 0, color: '#F5F5DC' }}>Wind Type</InputLabel>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <StyledSelect
-                            value={selectedWindType}
-                            input={<OutlinedInput/>}
-                            MenuProps= {{
-                                sx: {
-                                    '& .MuiPaper-root': {
-                                        backgroundColor: '#F5F5DC',
-                                    }
-                                }
+        <Grid container spacing={5} direction="column" classes={{ root: classes.transparentBackground }}>
+            <Grid item>
+                <FormControlLabel
+                    control={
+                        <Switch
```

Comment:
@syedomair0 Remove the Switch from Wind panel. You can directly show the "+ Add Wind Source" button.